chat_v2_sample/src/main/java/com/zendesk/chat_v2/sample/MainActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.zendesk.chat_v2.sample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import zendesk.chat.ChatEngine;
+import zendesk.messaging.MessagingActivity;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        if (Global.isMissingCredentials()) {
+            setContentView(R.layout.missing_credentials);
+            return;
+        }
+
+        MessagingActivity.builder()
+                .withEngines(ChatEngine.engine())
+                .show(this);
```

Comment:
Can we also showcase configuration here? Simply by passing ChatConfiguration with default params

Reply:
Ah yeah I guess that makes sense. I was trying to keep it as light as possible but probably makes sense to show it can be passed in.